src/classes/CRLP_Rollup_SEL.cls

```diff
@@ -265,11 +556,15 @@ public class CRLP_Rollup_SEL {
      * @param typeFilter Enum of the rollup filter to apply (All, DaysAgoOnly, NonDaysAgoOnly, etc.)
      * @return List<Rollup__mdt>
      */
-    public static List<Rollup__mdt> getRollupDefinitionsToUse(CRLP_RollupProcessingOptions.RollupType rollupType,
```

Comment:
- Uses more descriptive variable names
- Updated format of Boolean logic in `if` statements if more than one condition was present
    -  Enhances readability especially when there are many Boolean conditions
-  FYI: this method isn't used (or tested) anywhere

Reply:
it will be in another open work item.

Reply:
My comment is not true.  `CRLP_Rollup_SEL.getRollupDefinitionsToUse(...)` is used by other classes.